packages/common/src/codegen/render-solidity/userType.ts

```diff
@@ -1,25 +1,25 @@
 import { AbiTypeToSchemaType, getStaticByteLength, SchemaType, SchemaTypeToAbiType } from "@latticexyz/schema-type";
 import { StoreConfig, parseStaticArray } from "@latticexyz/config";
-import { RelativeImportDatum, RenderTableType } from "./types.js";
+import { RelativeImportDatum, RenderType } from "./types.js";
```

Comment:
I noticed in some areas we're able to remove the `.js` and others not, is this intentional?

Reply:
we should be able to remove it everywhere now!

Reply:
done in https://github.com/latticexyz/mud/pull/640/commits/77487c03bc674acebf1d7501810259c128e4a256